tinylog-api/src/main/java/org/tinylog/Logger.java

```diff
@@ -56,23 +59,44 @@ private Logger() {
 	 *
 	 * @param tag
 	 *            Tag for logger or {@code null} for receiving an untagged logger
+	 * @param otherTags
+	 *            Additional tags for the logger
 	 * @return Logger instance
 	 */
-	public static TaggedLogger tag(final String tag) {
-		if (tag == null || tag.isEmpty()) {
+	public static TaggedLogger tag(final String tag, final String...otherTags) {
```

Comment:
I would prefer two different methods: `tag(final String tag)` and `tags(final String... tags)`

Reply:
I was thinking if we went with the two methods, it would be possible to call the method with no tags (e.g. `Logger.tag().info(...)`) - something that wouldn't be possible with the single method. 

If we go with the two different methods, should we return the one from the `instance` field if its called with no tags? The other possibility I could think of is return a `TaggedLogger` with no tags, but I'm not sure how meaningful that is.

Reply:
Actually, `TaggedLogger` without tags is already in use by some users. In some use cases, an instance of tinylog's static `Logger` class is required. In this case, a `TaggedLogger` instance without any tags is the way to go.

I propose to see `Logger.tags()` as an alias for `Logger.tag(null)`. For me, it would be even legal to call `Logger.tags(null, "foo", "bar")`. In this case, the log entry will be logged as untagged log entry as well as tagged by `foo` and `bar`.

Reply:
That sounds good to me :)